app/controllers/pictures_controller.rb

```diff
@@ -0,0 +1,44 @@
+class PicturesController < ApplicationController
+
+  def index
+    @pictures = Picture.all
+  end
+
+  def new
+    @picture = Picture.new
+  end
+
+  def create
+    @picture = Picture.create(picture_params)
+    if @picture.save
+      redirect_to pictures_path
+    else
+      render 'new'
+    end
+  end
+
+  def picture_params
```

Comment:
Move this down to the bottom and make it private. This communicates that it's not a route.